js/src/bootstrap/rm.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const { getDescribe, getIt } = require('../utils/mocha')
+
+const expect = chai.expect
+chai.use(dirtyChai)
```

Comment:
If we are going to get `describe` and `it` through a local module, better to also pick `expect` from it, saving 4 lines from each module (and avoiding forgetting about it in the future).